docker/README.md

```diff
@@ -3,6 +3,7 @@
 Before you start, create a copy of the file `docker-compose.override.yml.example` and name it `docker-compose.override.yml`.
 Within this file you should configure the processes you want to test. 
 For the following processes, some example configurations have already been added to `docker-compose.override.yml.example`.
+If you want to use the `dic2-fhir-store-blaze` container, make sure to start the `keycloak` container first.
```

Comment:
Since `dic2-fhir-store-blaze` depends on `keycloak`, manual start is not necessary. If the reason for this comment is startup timing, please rephrase.

Reply:
The reason was indeed startup timing. Found a way to solve this by `depends-on` based on healtcheck:  8e58ab3